lib/common_test/src/ct_snmp.erl

```diff
@@ -224,13 +240,10 @@ start(Config, MgrAgentConfName, SnmpAppConfName) ->
     application:start(snmp),
 
     manager_register(StartManager, MgrAgentConfName).
- 
-%%% @spec stop(Config) -> ok
-%%%      Config = [{Key, Value}]
-%%%      Key = atom()
-%%%      Value = term()
-%%%
+
 %%% @doc Stops the snmp manager and/or agent removes all files created.
+-spec stop(Config) -> ok when
+      Config :: agent_config().
```

Comment:
This should not be agent_config(). It is the Config variable from the test case, defined as [{Key,Value}], where Key::atom(), Value::term().